Sources/VaporOAuth/Protocols/TokenManager.swift

```diff
@@ -1,21 +1,47 @@
 import Vapor
 
-public protocol TokenManager {
-    func generateAccessRefreshTokens(
+public protocol TokenManager: Sendable {
+    // Generates access, refresh, and ID tokens. Should be called after successful authentication.
```

Comment:
let's turn these into `///` documenation comments. They're helpful and we want people to see them!

Reply:
Done